pom.xml

```diff
@@ -129,11 +130,12 @@
 
         <!-- recommended installation versions -->
         <accumulo.version.recommended>1.9.3</accumulo.version.recommended>
-        <hbase.version.recommended>1.4.11</hbase.version.recommended>
+        <hbase.version.recommended>2.2.2</hbase.version.recommended>
+        <hbase.thirdparty.version.recommended>2.2.1</hbase.thirdparty.version.recommended>
         <cassandra.server.version.recommended>3.11.4</cassandra.server.version.recommended>
         <cassandra.driver.version.recommended>3.7.2</cassandra.driver.version.recommended>
         <hadoop.version.recommended>2.8.5</hadoop.version.recommended>
-        <zookeeper.version.recommended>3.5.6</zookeeper.version.recommended>
+        <zookeeper.version.recommended>3.4.14</zookeeper.version.recommended>
```

Comment:
3.4.14?  What happened to the 3.5.x line?

Reply:
the 3.5 line caused some compatibility issues with the default versions installed with hbase, etc... i think 3.4.14 you can just install and it works.

Reply:
Sweet...  any thoughts on this relative to https://geomesa.atlassian.net/browse/GEOMESA-2792 ?

Reply:
i'm pretty sure if there is an issue, it's due to being a much older 3.4.5 zk... it should not require zk 3.5. but we can verify when we fix that issue.